packages/client/src/__tests__/customerDataTriggers.test.ts

```diff
@@ -26,6 +26,12 @@ import { saul } from "@/__testData__/customers";
 
 const customersCollectionPath = `${Collection.Organizations}/${getOrganization()}/${OrgSubCollection.Customers}`;
 const bookingsCollectionPath = `${Collection.Organizations}/${getOrganization()}/${OrgSubCollection.Bookings}`;
+const publicOrgPath = `${Collection.PublicOrgInfo}/${getOrganization()}`;
+
+const publicOrgRef = getDocumentRef(
+  adminDb,
+  publicOrgPath
+);
```

Comment:
This seems unrelated to the file. If it's something you've forgotten to delete (after adding a different organization for calendar testing I imagine), please delete it. If not (and this was the only way to make this work), leave a `@TODO` comment to fix this more cleanly in the future (I'll do it with tests chore ticket)